### ISSUE TITLE
Not working with keytar 5

### ISSUE THREAD

**Issue by malammar** [jbreckmckye/electron-auth0-login#7]
The `optionalPeerDependency` defined in package.json:

```
  "optionalPeerDependencies": {
    "keytar": "4.3.0"
  }
```

results in this error when running `yarn start` with electron 7 and keytar 5 installed:

```
 App threw an error during load
Error: Version "5.0.0" of module "keytar" required by "electron-auth0-login" does not satisfy required range "4.3.0".
    at realRequire (c:\Users\Mustafa\dev\smart-margin-desktop\node_modules\codependency\index.js:184:9)
    at requirePeer (c:\Users\Mustafa\dev\smart-margin-desktop\node_modules\codependency\index.js:326:10)
    at Object.<anonymous> (c:\Users\Mustafa\dev\smart-margin-desktop\node_modules\electron-auth0-login\dist\index.js:20:16)
    at Module._compile (internal/modules/cjs/loader.js:880:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:892:10)
    at Module.load (internal/modules/cjs/loader.js:735:32)
    at Module._load (internal/modules/cjs/loader.js:648:12)
    at Module._load (electron/js2c/asar.js:717:26)
    at Function.Module._load (electron/js2c/asar.js:717:26)
    at Module.require (internal/modules/cjs/loader.js:775:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (c:\Users\Mustafa\dev\smart-margin-desktop\public\auth\auth.js:1:28)
    at Module._compile (internal/modules/cjs/loader.js:880:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:892:10)
    at Module.load (internal/modules/cjs/loader.js:735:32)
    at Module._load (internal/modules/cjs/loader.js:648:12)
```

I've tested electron-auth0-login with keytar 5/electron 7 and everything seems to work.

**Comment by jbreckmckye** [comment]
Okay, should be an easy upgrade. I'll do it tonight.

**Comment by jbreckmckye** [comment]
Released as `1.1.0` (https://www.npmjs.com/package/electron-auth0-login)

**Comment by malammar** [comment]
Works great. Thanks for the amazingly fast turnaround!
